X/XNNPACK/build_tarballs.jl

```diff
@@ -0,0 +1,72 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+using BinaryBuilderBase: os
+
+name = "XNNPACK"
+version = v"0.0.20200225"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/google/XNNPACK.git", "7493bfb9d412e59529bcbced6a902d44cfa8ea1c"),
+    DirectorySource("./bundled"),
+    GitSource("https://github.com/pytorch/cpuinfo.git", "d5e37adf1406cf899d7d9ec1d317c47506ccb970"),
```

Comment:
Sigh

Reply:
Yeah, I guess there's no way around it for these PyTorch dependencies...